### ISSUE TITLE
Fix Arithmetic tests on Databricks 14.3

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids#11526]
Build the plugin against Databricks 14.3 cluster using #11467. Once built successfully run the arithmetic tests by `TESTS=arithmetic_ops_test.py jenkins/databricks/test.sh` 

The following tests fail 
```
[gw1] [  2%] FAILED ../../src/main/python/arithmetic_ops_test.py::test_multiplication
[gw1] [  3%] FAILED ../../src/main/python/arithmetic_ops_test.py::test_multiplication_mixed
```

**Comment by razajafri** [comment]
Fixed by #11467

**Comment by razajafri** [comment]
Reopening as the draft branch hasn't been merged into the branch-24.12

**Comment by razajafri** [comment]
Fixed by #11635
